README.md

```diff
@@ -330,38 +330,43 @@ a step further by templating many files in a single run. The `template`
 function's `--help` provides examples:
 
 ```bash
-This will take an arbitrary number of source templates (or folders full
-of templates) and process them either in place (see --in-place) or into the
-folder specified as the last argument. It will make any folders required
-along the way. If a source is an existing file (not a folder) it will be
-treated as a template regardless of the extension (though if the extension
-matches it will still be removed).
+$ go run . template --help
```

Comment:
I don't think there is any value in the readme showing how to run the `--help`

Reply:
[Fixed](https://github.com/pastdev/clconf/pull/13/commits/e86bd2cdf541c247e3007ed37169507d2d56a096)